lib/slack.js

```diff
@@ -138,14 +138,14 @@ module.exports = class Slack {
     this.controller.on('direct_message', (bot, message) => this.handleDirectMessage(message));
   }
 
-  static createIssueAttachment(issue) {
+  static createIssueAttachment(issue, color) {
```

Comment:
`static createIssueAttachment(issue, params) {` would make for a more stable signature. Subsequently, we'd use `return { ...params, [other stuff] }` down below.